### ISSUE TITLE
Hub link is not intuitive on mobile

### ISSUE THREAD

**Issue by patricklafrance** [retypeapp/retype#625]
Hello!

While on mobile, since the logo and the hub link are centered, the hub link purpose is harder to grasp for the user.

![hey](https://github.com/retypeapp/retype/assets/794579/1dcc084e-8473-488b-ba17-2ffa56514193)

I am not sure what would be the ideal UI.[ React Spectrum](https://react-spectrum.adobe.com/) solve this by moving the logo and the hub link inside the left sidebar when on mobile:

![image](https://github.com/retypeapp/retype/assets/794579/8eae930a-a0fb-427a-8e83-35318f45ff31)

Personally, I like React Spectrum solution because it free a lot of spaces when on mobile and makes the UI less loaded. But it's not compatible with the current Retype mobile design, because there's already an arrow in the left sidebar:

![image](https://github.com/retypeapp/retype/assets/794579/680f0564-a001-4418-97d3-155b787a8e5c)

Let me know what you think!

Thank you,

Patrick

**Comment by patricklafrance** [comment]
When on mobile, the title and logo could be moved inside the mobile menu:

![Image](https://github.com/user-attachments/assets/3a65d92d-2e5a-4704-a4ac-f15459071e60)

![Image](https://github.com/user-attachments/assets/610b9766-6277-40e4-84e1-ec2101cd0155)

Then current back arrow could either be removed or moved to the right of the menu maybe with a cross icon instead:

![Image](https://github.com/user-attachments/assets/85a80d6a-e9f1-4e3d-a71f-204d7674528a)

That would leave some space for the actual back arrow becoming the arrow navigating to the hub when activated:

![Image](https://github.com/user-attachments/assets/a61e70fc-a43b-4069-ac89-a9665f2bd6b9)

Let me know what you think!

Patrick

**Comment by geoffreymcgill** [comment]
Agreed. We definitely need to adjust the mobile layout. I have tagged for the 3.11.0 release. I cannot guarantee we will implement within the 3.11.0 release, but it's on our radar and we will try our best to get this resolved soon.
